ntbs-service/Pages/Notifications/Edit/HospitalDetails.cshtml.cs

```diff
@@ -108,8 +113,10 @@ protected override IActionResult RedirectAfterSaveForDraft(bool isBeingSubmitted
 
         protected override async Task ValidateAndSave()
         {
+            FormattedNotificationDate.TryConvertToDateTime(out var notificationDate);
+            var notificationDateHasChanged = notificationDate.HasValue && notificationDate != Notification.NotificationDate;
```

Comment:
Are there any validation implications from these? If not could we move it into ModelState.IsValid scope?